.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,16 @@
-_Please note that while we are transitioning from travis-ci to AWS CodeBuld, some tests are run on each platform. Non-AWS contributors will temporarily be unable to see CodeBuild results. We apologize for the inconvenience._
+_Please note that while we are transitioning from travis-ci to AWS CodeBuild, some tests are run on each platform. Non-AWS contributors will temporarily be unable to see CodeBuild results. We apologize for the inconvenience._
 
-**Issue # (if available):** 
+### Summary of changes: 
```

Comment:
i think i would still prefer "Description" over "Summary" because it leaves to author to decide how detailed or summarized the description to be

Reply:
Changed.